src/mkdocs_git_revision_date_localized_plugin/util.py

```diff
@@ -92,8 +92,15 @@ def get_git_commit_timestamp(
                     commit_timestamp = commit_timestamp.split()[-1]
             else:
                 # Latest commit touching a specific file
+                if self.config.get('last_update_exclude_renames'):
+                    # We can remove if-else statement and do a one-liner as well
+                    diff_filter_param = "r"
+                    follow_param = True
+                else:
+                    diff_filter_param = ""
+                    follow_param = False
                 commit_timestamp = git.log(
-                    realpath, date="unix", format="%at", n=1, no_show_signature=True
+                    realpath, date="unix", format="%at", diff_filter=diff_filter_param, n=1, no_show_signature=True, follow=follow_param
```

Comment:
Note that with follow enabled, you will get multiple commits, and you need to take the last one. See the logic used for the creation date.

Reply:
Doesn't `n=1` also achieve the same desired effect as selecting the last one using `commit_timestamp = commit_timestamp.split()[-1]` ?

I saw that and though maybe it is just awaiting refactoring.

If it doesn't achieve the same result, I can do it like the creation date.